image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       affinity:
         nodeAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
-            # Central is single-homed, so avoid preemptible nodes.
+            # Central is single-homed, so avoid preemptive nodes.
```

Comment:
Why this change? The key is `.../gke-preemptible`, not `...preemptive`.

Reply:
Oops, I thought it was a typo